src/components/connect.js

```diff
@@ -153,11 +156,23 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
 
       refetchDatum(prop, mapping) {
         const startedAt = new Date()
+        const isRefresh = mapping.refreshInterval > 0
 
         if (this.state.refreshTimeouts[prop]) {
           window.clearTimeout(this.state.refreshTimeouts[prop])
         }
 
+        // do not refresh while tab is hidden and refreshHidden is unset
+        if (isRefresh && !mapping.refreshHidden && visibility.hidden()) {
```

Comment:
It's a corner case, but it would probably be good to also check for `this.state.mappings[prop] && this.state.mappings[prop].fulfilled` so that this does not block the initial load in case the tab is hidden.